### PR TITLE
Adapterjava

### PR DESCRIPTION
**Pull request by pantianying**

### CODE REVIEW

**Comment by DogBaoBao** [comment]
Add this code, the example use http request to dubbogo proxy reports to dubbogo provider is ok.

File: dubbo/java.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package dubbo
+
+import (
```
Comment (AlexStocks) [inline dubbo/java.go:3]: split

File: go.mod [hunk 2]
```diff
@@ -5,11 +5,11 @@ go 1.12
 require (
 	github.com/apache/dubbo-go v1.4.0-rc1
 	github.com/apache/dubbo-go-hessian2 v1.4.0
-	github.com/dubbogo/getty v1.3.3
+	github.com/dubbogo/getty v1.3.9
```
Comment (AlexStocks) [inline go.mod:8]: github.com/apache/dubbo-getty

**Comment by DogBaoBao** [comment]
this is old rev, not used.

**Comment by DogBaoBao** [comment]
this is old rev, not used.

**Pull request closed without merging**